app/src/main/java/com/example/tmdb/screens/home/Home.kt

```diff
@@ -163,17 +181,21 @@ fun <T : Any> TabsContent(
     count: Int,
     list: LazyPagingItems<T>,
```

Comment:
As I can see, every place you're calling TabsContent has this `T` as `Movie` class.

Because you've created a function generic you forced yourself to use reflection in the `RowItem`.

Please refactor this, reflection is a big no :D

Reply:
I thought it was good to use reflection because I wanted to avoid "duplicating" methods because I am currently calling those methods for tv class and movie class otherwise I would have to call a separate method for tv especially for movie. So if that double call doesn't bother you, then I'll refactor it.